modules/users/client/views/authentication/signin.client.view.html

```diff
@@ -72,7 +72,8 @@
               </div>
             </fieldset>
           </form>
-          <div class="text-white">
+          <!-- For bigger screens -->
+          <div class="text-white hidden-hs">
```

Comment:
It is class hidden-xs
Now it is still displaying both

Reply:
7a1b22f6ae92ffc04df004b818f2341ce4ce5c2c